ga_resources/views/__init__.py

```diff
@@ -5,7 +5,8 @@
 from django.contrib.auth.models import Group
 import importlib
 from django.contrib.gis.geos import Polygon, GEOSGeometry
-from django.db.models import get_model
+from django.apps import apps
+get_model = apps.get_model
```

Comment:
`get_model` is used within the file (line 224).
This perhaps would have been more apparent if I broke this up differently.

line 8: `from django.apps import apps`
line 224: `model = apps.get_model(request.POST["app"], request.POST["model"])`

Same could be applied to other files using `get_model` or `get_models`